lib/mail_room/mailbox.rb

```diff
@@ -30,47 +35,44 @@ module MailRoom
       :ssl => true,
       :start_tls => false,
       :delete_after_delivery => false,
-      :delivery_options => {}
+      :delivery_options => {},
+      :arbitration_method => 'noop',
+      :arbitration_options => {}
     }
 
     # Store the configuration and require the appropriate delivery method
     # @param attributes [Hash] configuration options
     def initialize(attributes={})
       super(*DEFAULTS.merge(attributes).values_at(*members))
+    end
 
-      require_relative("./delivery/#{(delivery_method)}")
+    def delivery
+      @delivery ||= Delivery[delivery_method].new(parsed_delivery_options)
     end
 
-    # move to a mailbox deliverer class?
-    def delivery_klass
```

Comment:
Could this remain, simply as a method wrapper around `Delivery[delivery_method]`?